lms/djangoapps/student_account/views.py

```diff
@@ -293,3 +303,63 @@ def _external_auth_intercept(request, mode):
         return external_auth_login(request)
     elif mode == "register":
         return external_auth_register(request)
+
+
+@login_required
+@require_http_methods(['GET'])
+def account_settings(request):
+    """Render the students account settings page.
+
+    Args:
+        request (HttpRequest)
+
+    Returns:
+        HttpResponse: 200 if the page was sent successfully
+        HttpResponse: 302 if not logged in (redirect to login page)
+        HttpResponse: 405 if using an unsupported HTTP method
+
+    Example usage:
+
+        GET /account/settings
+
+    """
+    released_languages = DarkLangConfig.current().released_languages_list
+
+    # add in the default language if it's not in the list of released languages
+    if settings.LANGUAGE_CODE not in released_languages:
+        released_languages.append(settings.LANGUAGE_CODE)
+        # Re-alphabetize language options
+        released_languages.sort()
+
+    language_options = [language for language in settings.LANGUAGES if language[0] in released_languages]
+
+    country_options = [
+        (country_code, unicode(country_name))
+        for country_code, country_name in sorted(
+            countries.countries, key=lambda(__, name): unicode(name)
+        )
+    ]
+
+    context = {
```

Comment:
I'm thinking that there should be tests that the correct set of choices are presented for each field. You could do this in Bok Choy but it would be very laborious. One possibility would be to factor the creation of this context into a separate method and then write unit tests for it.